Library/Homebrew/utils/gems.rb

```diff
@@ -144,9 +144,7 @@ def install_bundler_gems!(only_warn_on_failure: false, setup_path: true)
 
       # for some reason sometimes the exit code lies so check the output too.
       if bundle_check_failed || bundle_check_output.include?("Install missing gems")
-        begin
-          safe_system_redirect_stdout_to_stderr bundle, "install"
-        rescue ErrorDuringExecution
+        unless system bundle, "install"
```

Comment:
@gibfahn `safe_system_redirect_stdout_to_stderr` isn't required here so this doesn't work unfortunately. It can't be required, either. I think `$stdout.reopen($stderr)` or similar should be inlined in both locations instead.

Reply:
Oh dear, I must not have been testing that code path after I changed it.

Fair enough on the suggestion, will do so.